python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/test_resources.py

```diff
@@ -205,3 +210,65 @@ def _mock_interaction():
     else:
         with pytest.raises(Failure, match="failed!"):
             _mock_interaction()
+
+
+def test_fivetran_materialization(
+    connector_id: str,
+    fetch_workspace_data_api_mocks: responses.RequestsMock,
+    sync_and_poll: MagicMock,
+) -> None:
```

Comment:
nit: mention that this is actually testing sync_and_poll. Calling it the materialization method feels a bit overly specified to me.